subsys/bootloader/bl_storage/bl_storage.c

```diff
@@ -72,6 +72,9 @@ uint32_t num_public_keys_read(void)
 	return nrfx_nvmc_uicr_word_read(&BL_STORAGE->num_public_keys);
 }
 
+
+#ifdef CONFIG_SECURE_BOOT_CRYPTO
```

Comment:
I don't understand this part about conditional compilation exlusion of these functions.
They appear to all be about modifying or reading what is stored, and no crypto appears to be involved.

Reply:
They rely on definitions (`SB_*`) only available when bl_crypto is enabled, see e.g. line 90.

Reply:
I can only see SB_PUBLIC_KEY_HASH_LEN, which does not appear to have any conditionals on it.
It's also used in the definition of `struct bl_storage_data`, which is the struct definition for BL_STORAGE.

Am I overlooking something?

Reply:
I removed these, seems to work just fine.